client/scripts/views/components/header/login_selector.ts

```diff
@@ -167,23 +174,33 @@ const LoginSelector: m.Component<{
 
             if (originAddressInfo) {
               try {
+                const targetChain = joiningChain || originAddressInfo.chain;
+                const currentPrefix = app.config.chains.getById(targetChain).ss58Prefix;
```

Comment:
Is there any reason we're using the SS58Prefix from the DB here rather than the one fetched from the chain? The two should be identical, of course, but we may want to remove the latter if we're going to be storing the database's version.

Reply:
Because the chain is not initialized on every pages. For example, on members page and on profile page, the chain is not initialized and I can't get ss58 prefix from the chain info.

Reply:
In that case, we may want to use the database's SS58 number as the source for the entire codebase, and remove the one fetched from chain, so it's not confusing as to why we have two separate SS58 numbers stored... But up to you whether this is worth doing now or later.